controllers/articles.js

```diff
@@ -29,15 +29,59 @@ router.get('/new', (req, res) => {
 })
 
 // GET /articles/:id - display a specific post and its author
+// router.get('/:id', (req, res) => {
+//   db.article.findOne({
+//     where: { id: req.params.id },
+//     include: [db.author]
+//   })
+//   .then((article) => {
+//     if (!article) throw Error()
+//     // console.log(article.author)
+//     res.render('articles/show', { article: article })
+//   })
+//   .catch((error) => {
+//     console.log(error)
+//     res.status(400).render('main/404')
+//   })
+// })
+
+// GET route /article/id
 router.get('/:id', (req, res) => {
   db.article.findOne({
     where: { id: req.params.id },
     include: [db.author]
   })
   .then((article) => {
     if (!article) throw Error()
-    console.log(article.author)
-    res.render('articles/show', { article: article })
+    db.comment.findAll({
+      where: {articleId: req.params.id}
+    })
+    .then((comments) => {
+      console.log('comments')
+      console.log(comments)
+      res.render('articles/show', { article: article, comments: comments })
+    })
+    .catch((error) => {
+      console.log(error)
+      res.status(400).render('main/404')
+    })
+  })
+  .catch((error) => {
+    console.log(error)
+    res.status(400).render('main/404')
+  })
+})
+
+// POST route for comments
+router.post('/comment', (req, res) => {
+  console.log('req.body')
+  console.log(req.body)
+  db.comment.create(
+    req.body
+  )
```

Comment:
I think Adam talked about using req.body in class yesterday.  How did this work for you?  I always use the more explicit syntax, but from what I remember, this should work!